docs/source/pySDC/core.rst

```diff
@@ -52,6 +52,14 @@ core.Level module
    :undoc-members:
    :show-inheritance:
 
+core.Nodes module
```

Comment:
I guess sphinx created all these things automatically. I can remove them from the commit, but I didn't know what to do with them